resources/skins/nudus-base/html_style_base.scss

```diff
@@ -18,11 +18,9 @@ $base-unit:     10px !default;
 //
 
 // Let the font be customised via RSS Guard settings
-// Note: Font size there related **only** to that font alone, it is
-// not absolute, and nothing else can be done from my side
-// E.g. "Roboto 10" <-- something like this is send from RSS Guard side
 * {
     font-family: inherit;
+    font-size: inherit;
```

Comment:
Please note that even though I added this line, it appears to change nothing at all.

Added it just to make sure.